docs/storage/lumio/clients-general.md

```diff
@@ -0,0 +1,424 @@
+# Transferring and managing data
```

Comment:
I would use "Managing data" as the name for this page. It makes menu navigation easier.